### PR TITLE
Remove `sdk.getComponent()` where possible

### PR DESCRIPTION
**Pull request by SimonBrandner**
Type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://61863ff859398be616ba6ab1--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/dialogs/InfoDialog.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default class InfoDialog extends React.Component<IProps> {
     };
 
     render() {
+        // FIXME: Using a regular import will break the app
```
Comment (t3chguy) [inline src/components/views/dialogs/InfoDialog.tsx:47]: shouldn't this comment remain?

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/dialogs/InfoDialog.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/components/views/dialogs/InfoDialog.tsx:47]: _Remain_? The line is being added not being removed, what am I missing? :sweat_smile:

Comment (t3chguy) [reply src/components/views/dialogs/InfoDialog.tsx:47]: oh dear, first PR review in the morning :')

**Pull request merged**